### PR TITLE
Add nsenter

### PR DESCRIPTION
**Pull request by pabateman**

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/nsenter.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: nsenter
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/pabateman/kubectl-nsenter
+  shortDescription: "Execute shell command under container namespace of pod"
```
Comment (ahmetb) [inline plugins/nsenter.yaml:8]: may we rewrite this as:

> Run commands in Pod's namespace on the node over SSH

File: plugins/nsenter.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  description: |
+    Kubectl plugin for pod's linux namespaces command execution via direct node ssh connection
```
Comment (ahmetb) [inline plugins/nsenter.yaml:10]: This is not very descriptive, can you rephrase this like:

> This plugin establishes a connection to node that's running the Pod over SSH and uses
> nsenter to run commands in the container's namespace. You would need something
> like this to run programs that are missing in the container image.

not verbatim but your own summary.

File: plugins/nsenter.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  caveats: |
+    Almost done. Now activate your ssh agent ang go on!
```
Comment (ahmetb) [inline plugins/nsenter.yaml:12]: This is not quite helpful, you can use this field to list some relevant caveats that could be show-stoppers to your plugin:

```
* This plugin needs SSH access to nodes.
* Your nodes need to have `nsenter` command.
```

**Review by ahmetb — CHANGES_REQUESTED** [review]
Quite a clever and helpful plugin with an elegant implementation. Left some comments for you to address.
/approve

File: plugins/nsenter.yaml (hunk 3 above)
Comment (pabateman) [reply plugins/nsenter.yaml]: @ahmetb thanks for the feedback and review! Agree with all comments and corrected manifest according to it, with my own vision ofc :)

**Comment by pabateman** [comment]
@ahmetb thanks for the feedback and review! Agree with all comments and corrected manifest according to it, with my own vision ofc :)

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!

Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!

Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.
